src/Queue/QueueServiceProvider.php

```diff
@@ -108,19 +107,17 @@ public function __call($name, $arguments)
         $this->registerCommands();
     }
 
-    protected function registerCommands()
+    private function registerCommands()
     {
-        $this->app['events']->listen(Configuring::class, function (Configuring $event) {
-            $queue = $this->app->make(Queue::class);
+        $queue = $this->app->make(Queue::class);
 
-            // There is no need to have the queue commands when using the sync driver.
-            if ($queue instanceof SyncQueue) {
-                return;
-            }
+        // There is no need to have the queue commands when using the sync driver.
+        if ($queue instanceof SyncQueue) {
+            return;
+        }
```

Comment:
Can you reset the changes you made please so that the PR only reflects the changes. Otherwise we will merge changes that do nothing but rewrite the same code, thank you.